### DISCUSSION TITLE
《Go 语言设计与实现》纸质书预售了！ - 面向信仰编程 · /golang-book-release

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#650] [Announcements]
https://draveness.me/golang-book-release

**Comment by zcong1993** [comment]
已买，博客忠实读者必须支持一波👍

**Comment by leafduo** [comment]
已下单三本，一本观赏用，一本保存用，一本布教用

**Comment by MoonShining** [comment]
当然要支持了！

**Comment by draveness** [comment]
> @leafduo 已下单三本，一本观赏用，一本保存用，一本布教用

这也太夸张了

**Reply by tony9527167** [reply]
再买一本，垫显示器 XD

**Comment by OhYee** [comment]
坏了，没有第一时间看到这篇博客……

下单了，不知道还有没有签名版🥲

**Comment by draveness** [comment]
> @OhYee  坏了，没有第一时间看到这篇博客……
> 
> 下单了，不知道还有没有签名版🥲

其实昨晚上才发，不过你能下单就是还有

**Comment by kylinzhu** [comment]
深圳无货

**Comment by draveness** [comment]
> @kylinzhu 深圳无货

已经补货了

**Comment by wujunwei** [comment]
已下单，支持博主！！

**Comment by BaoXuebin** [comment]
下单支持👍

**Comment by linw1995** [comment]
已下单支持。博主有考虑卖 PDF 版的吗

**Comment by aimyon36** [comment]
生活费全部买鞋了，下个月再说吧🤣🤣🤣

**Comment by automano** [comment]
怕不是已经售罄了

**Comment by 0000CAFE** [comment]
刚刚看到，赶紧下单一本学习

**Comment by louiclille** [comment]
以后用德莱文跟上单对线时我决定买一本放桌上试试运气

**Comment by vettelx** [comment]
已经没有签名版了！

**Comment by baici1** [comment]
显示正在采购，会有签名版吗？感觉在追星🤣✌！

**Comment by dovefi** [comment]
已购

**Comment by arlottang** [comment]
已买

**Comment by CNife** [comment]
没货啦，希望赶紧补货

**Comment by longfei1992** [comment]
深圳显示无货，要12月8号发

**Comment by pengpaige** [comment]
已下单, 支持一下博主.

**Comment by woofyzhao** [comment]
支持大佬，已下单

**Comment by BillZong** [comment]
加班没注意博客通知，深圳又无货了~
可惜了大佬的签名版啊 ORZ

**Comment by HonlinRen** [comment]
已买，表示支持~！

**Comment by RTsien** [comment]
买书送德莱文皮肤吗

**Comment by BrianQy** [comment]
！！！不出意外 这周能拿个offer 要转go！ 刚好巧了！支持！感谢博主的文章对我面试的帮助！（。。。刚发现没货了。。）

**Comment by ycxb0315** [comment]
刚看到没货了，希望大佬补货，期待中

**Comment by ExcitingFrog** [comment]
昨天就无了  今天看还是无了 好快
加点 加点

**Comment by csbo98** [comment]
emmm，上海无货

**Comment by clcy1243** [comment]
限购了吗？只能买一本，还想送朋友两本来着

**Comment by otailai** [comment]
上海什么时候补货啊大佬

**Comment by MowenPan** [comment]
没货了哇，7折活动已经结束了吗？现在京东上的价格是104.9

**Comment by wujunwei** [comment]
> @leafduo
> 已下单三本，一本观赏用，一本保存用，一本布教用

到货了吗？我22号下单，现在还没发货

**Comment by leafduo** [comment]
> > @leafduo
> > 已下单三本，一本观赏用，一本保存用，一本布教用
> 
> 到货了吗？我22号下单，现在还没发货

这个是预售，12 月 8 号左右（好像是之前）发货

**Comment by iDvel** [comment]
来晚了吗，北京无货。

**Comment by nonbutAworker** [comment]
签名版没了啊，能补货吗？我推荐补充1W本签名版~！

**Comment by draveness** [comment]
> @nonbutAworker  签名版没了啊，能补货吗？我推荐补充1W本签名版~！

签名版已经卖光了，京东上普通版上架了

**Comment by atmoxao** [comment]
> @MowenPan
> 没货了哇，7折活动已经结束了吗？现在京东上的价格是104.9

现在是 8.9折 124
---
我的我的，没成想到买的是签名版。

**Comment by OhYee** [comment]
返图😄

![IMG_20211202_111111](https://user-images.githubusercontent.com/13498329/144351252-4c0245df-61d9-40ae-a220-41c1d676eb04.jpg)

**Comment by leafduo** [comment]
我的也到啦！

![IMG_1305](https://user-images.githubusercontent.com/28619/144371451-888791d5-ceed-40aa-8440-4c066ddd4717.jpg)

**Comment by PudongLi** [comment]
已收到，孩子很爱吃👍🏼

**Comment by fengbianyun** [comment]
已经收到。谢谢左大的签名

**Comment by lxdlam** [comment]
我也收到了，支持左大！

**Comment by guowei-gong** [comment]
已支持

**Comment by 0000CAFE** [comment]
到货了，上面有左大的签名，哈哈哈

**Comment by dadait** [comment]
买了纸质版，色彩搭配让人阅读起来感觉很舒适。

**Comment by Xpectuer** [comment]
已到货，当作博客的二刷
![IMG_1250](https://user-images.githubusercontent.com/43370918/146714067-977d13ac-e37f-4d90-8a30-b768a3054d9f.JPG)

**Comment by love477** [comment]
大佬去shopee了？

**Comment by oliver-com** [comment]
估计是赶不上签名版了

**Comment by Cherrison** [comment]
大佬run回国了?

**Comment by fbaiesec** [comment]
已支持，静等书籍到，好好学习

**Comment by fire1220** [comment]
已买，讲的很好

**Comment by alexhuang19940623** [comment]
想问下，有没有电子版，不在国内，但是从事go开发，想买电子版

**Comment by sammyluck** [comment]
去年已买实体书，听说今年下半年 Go 语言设计和实现 会出修订版，请问下大概什么时候出？

**Reply by draveness** [reply]
暂无

**Comment by maoturing** [comment]
有上架微信读书的打算吗?

**Reply by draveness** [reply]
这个也暂时没有

**Comment by gaojs** [comment]
限量签名版的二维码，可以撤了。因为扫了，也跳转不到了。

**Comment by dnxxymcj** [comment]
大佬们，有入门级的书推荐吗
